documentation/ingestion_server/guides/mapping.md

```diff
@@ -0,0 +1,60 @@
+# Mapping database tables to Elasticsearch
```

Comment:
This page in general feels like it might not be that useful. Specifically, I think the relevant parts of it could be moved into the module docstring for `elasticsearch_models.py` and some of it plainly duplicates documentation that is already present in the code next to the stuff being documented (and is therefore more likely to be maintained and updated). It would be good for us to avoid documenting specific implementation details if possible.